datadog_cluster_agent/datadog_checks/datadog_cluster_agent/check.py

```diff
@@ -43,6 +43,8 @@
     'autodiscovery_poll_duration': 'autodiscovery.poll_duration',
     'autodiscovery_watched_resources': 'autodiscovery.watched_resources',
     'autodiscovery_errors': 'autodiscovery.errors',
+    'kubernetes_apiserver_kube_events': 'kubernetes_apiserver.kube_events',
+    'kubernetes_apiserver_emitted_events': 'kubernetes_apiserver.emitted_events',
```

Comment:
Could you also add them to the tests [here](https://github.com/DataDog/integrations-core/blob/master/datadog_cluster_agent/tests/test_datadog_cluster_agent.py#L15)?